microsoft/testsuites/core/storage.py

```diff
@@ -345,6 +345,42 @@ def hot_add_disk_parallel_premium_ssd(self, log: Logger, node: Node) -> None:
             log, node, DiskType.PremiumSSDLRS, self.DEFAULT_DISK_SIZE_IN_GB
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case will verify mount azure nfs 4.1 on guest successfully.
```

Comment:
Where is it specified to 4.1? Maybe it's upgraded to 4.2 later?

Reply:
now not specify the version, by default it is 4.1